src/status_im/subs/wallet/send.cljs

```diff
@@ -34,6 +34,11 @@
  :<- [:wallet/wallet-send]
  :-> :token)
 
+(rf/reg-sub
```

Comment:
I believe we should have a :symbol within `[:wallet :ui :send :token]`. Isn't it the same thing?

Reply:
Sometimes we only have `:token-symbol`, there's an old comment that explains the reason -> https://github.com/status-im/status-mobile/blob/74f1281631d148680bf35dde090aad934ecf5002/src/status_im/contexts/wallet/send/events.cljs#L152-L154